engine/pokedex/pokedex_3.asm

```diff
@@ -1,3 +1,5 @@
+	ret ; unused
```

Comment:
I added it to pokedex_3.asm because that way it's grouped with other code, and plausibly they could have dummied out some SGB or prototype routine there. But sure.